### DISCUSSION TITLE
use hylang

### DISCUSSION THREAD

**Discussion by wdduncan** [hsolbrig/funowl#51] [Ideas]
@hsolbrig I enjoyed your presentation at ICBO. Some of your examples seemed like they might be more easily implemented in a LISPy syntax. Have you considered using [hylang](https://github.com/hylang/hy)?   

I know ... it's a crazy suggestion. But, I thought I'd throw it out there :)

**Comment by hsolbrig** [comment]
@wdduncan Sounds like a really interesting idea.  The bit that we need to implement this would be a s-expression specification for OWL stated in terms of the functional spec.  I think that it exists.  If so, lets prototype a couple of hylang to functional macro definitions to make sure we've got a handle on it and maybe we can automate the rest?

Happy to chat via zoom or similar...

**Comment by wdduncan** [comment]
@hsolbrig Sure. Happy to explore. I can ping you on Skype :)
